twatbot.py

```diff
@@ -265,8 +270,8 @@ def htmlize_twat(twat, vars):
 					href = i
 					title = "view remote image"
 				elif 'video' in twat or 'ext_tw_video_thumb' in i:
-					if os.path.exists('users/%s/%s.mp4' % (twat['user'].lower(), str(twat['id']))):
```

Comment:
maybe we should have a mp4_path variable just like the img_path above (to replace the 2 copies of the path-constructing code)